### PR TITLE
upgraded

### PR DESCRIPTION
**Pull request by developer0623**

### CODE REVIEW

**Comment by umurb** [comment]
@developer0623 can you please fix: 

1. Send Confirmation modal breaks down to a new line if the input is long; e.g. 200.000001

screen shot 2018-08-21 at 12 42 02 pm

2. Floating labels in Add Custom Node modal are not working properly
floating-label-error

**Comment by developer0623** [comment]
@umurb It seems the riot server was down.
I just fixed the `AddNodeModal`
Kindly check

File: app/components/AddDelegateModal/index.js [hunk 1]
```diff
@@ -40,17 +41,20 @@ type Props = {
   managerBalance: number
 };
 
+const InputAddressConainer = styled.div`
```
Comment (umurb) [inline app/components/AddDelegateModal/index.js]: InputAddressContainer

File: app/components/AddDelegateModal/index.js [hunk 2]
```diff
@@ -320,46 +334,33 @@ class AddDelegateModal extends Component<Props> {
...
-          onClick={this.onCloseClick}
-        />
-        <InputAddress
-          labelText={t('general.delegate_address')}
-          addressType="delegate"
-          tooltip
-          changeDelegate={this.changeDelegate}
-          onIssue={(status)=> this.setState({isDelegateIssue: status})}
-        />
+        <InputAddressConainer>
```
Comment (umurb) [inline app/components/AddDelegateModal/index.js]: InputAddressContainer*

File: app/components/CreateAccountSlide/InputValidComponent.js [hunk 3]
```diff
@@ -130,9 +107,10 @@ export default class InputValidComponent extends Component<Props> {
 
   render() {
     const label = this.getLabel(this.props.index);
+    const {errorText} = this.state;
     return (
       <StyledInputContainer>
-        <TextField
+        {/* <TextField
```
Comment (umurb) [inline app/components/CreateAccountSlide/InputValidComponent.js]: Why comment out and not delete?

**Pull request merged**